app/src/main/java/org/schabi/newpipe/util/Localization.java

```diff
@@ -132,14 +137,14 @@ public static String localizeViewCount(Context context, long viewCount) {
         return getQuantity(context, R.plurals.views, R.string.no_views, viewCount, localizeNumber(context, viewCount));
     }
 
-    public static String localizeSubscribersCount(Context context, long subscriberCount) {
```

Comment:
Since this function is exactly the same as `shortSubscriberCount`, wouldn't it be better to rename `shortSubscriberCount`  to `localizeSubscriberCount`? The output of the function is a normal (i.e. with no attributes about size) output, since it can't be considered small when comparing it to the normal output if the normal output does not exist ;-) 

###### Note: removed `s` from subscribers

Reply:
In the first commit
```
Subscriber count is always giving the short count.
We can't get exact number since this YouTube update: https://support.google.com/youtube/thread/6543166
But only short count (B, M, k), so showing full number, eg for 1.9M: 1,900,000, is wrong because the number could be 1,923,490 or 1,897,789…
```
I sticked to this class «convention»
shortSOMETHINGcount is giving the number with the abbreviation at the end (eg 5000000-> 5M in English US)
localizeSOMETHINGcount is formatting the number (eg 5 million is 5,000,000 in English US but 5 000 000 in French).

So no, this is not the same. 5,000,000 != 5M.

For the s, I followed localizeViewCount (not views count)

Then I don't really understand your comment. Can you re-explain please?

Reply:
Oh, my fault, I misunderstood something, this is good 😅